### DISCUSSION TITLE
RAPIDS - manifest for docker not found

### DISCUSSION THREAD

**Discussion by sureshksampath** [carissalow/rapids#210] [Q&A]
Hello,

I'm getting an error when I'm trying to pull the RAPIDS docker container on a Linux RHEL server where the docker desktop is available. Does the docker image still exist? Please advise.

$ docker pull moshiresearch/rapids:latest
Trying to pull repository registry.access.redhat.com/moshiresearch/rapids ...
Pulling repository registry.access.redhat.com/moshiresearch/rapids
Trying to pull repository registry.redhat.io/moshiresearch/rapids ...
Trying to pull repository docker.io/moshiresearch/rapids ...
Trying to pull repository docker.io/moshiresearch/rapids ...
manifest for docker.io/moshiresearch/rapids:latest not found

Regards,
Suresh

**Comment by JulioV** [comment]
Hi, I was able to pull the latest image in my mac. You can find it here https://hub.docker.com/r/moshiresearch/rapids/tags

@jenniferfedor if you guys have access to a linux machine or vm could you try pulling it?

In the meantime I would try reinstalling docker desktop @sureshksampath

**Comment by sureshksampath** [comment]
Hello, I was able to pull the docker image after the latest version of Docker community edition 20.10
23 was installed, and able to successfully complete the installation. It didn't work with the older version of Docker.

Thank so much for your help!
Suresh

________________________________

**Comment by jenniferfedor** [comment]
Hi @sureshksampath, I'm glad to hear that updating Docker resolved this issue! Thank you again for reporting.  

Hi @JulioV, I was also able to pull our Docker image without any problems. As part of our next release, should I update our RAPIDS Docker installation instructions to specify a specific version of Docker/recommend upgrading to the latest version? Thank you!

**Reply by JulioV** [reply]
@jenniferfedor yes, please
